translations/harbour-barcode-pt_PT.ts

```diff
@@ -0,0 +1,378 @@
+        <extracomment>Switch button description (explanation)</extracomment>
+        <translation>Isto vai apagar todas as imagens de código de barras guardadas anteriormente.</translation>
+    </message>
+    <message id="settings-save_images-confirm_delete">
+        <source>Confirm delete</source>
+        <extracomment>Button label (confirm deletion of image files)</extracomment>
+        <translation>Confirmar eliminação</translation>
+    </message>
+    <message id="settings-history-section">
+        <source>History</source>
+        <extracomment>Section header</extracomment>
+        <translation>Historial</translation>
+    </message>
+    <message id="settings-history-slider_label">
+        <source>Max history size (saved values: %1)</source>
+        <extracomment>Slider label</extracomment>
+        <translation>Tamanho máx. do historial (guardados: %1)</translation>
+    </message>
+    <message id="settings-value-deactivated">
+        <source>deactivated</source>
+        <extracomment>Generic slider value</extracomment>
+        <translation>desativado</translation>
+    </message>
+    <message id="settings-history-slider_value" numerus="yes">
+        <source>%1 item(s)</source>
+        <extracomment>History slider value</extracomment>
+        <translation>
+            <numerusform>%1 elemento</numerusform>
+            <numerusform>%1 elementos</numerusform>
+        </translation>
+    </message>
+    <message id="settings-history-confirm_resize">
+        <source>Confirm resize</source>
+        <extracomment>Button label</extracomment>
+        <translation>Confirmar redimensionamento</translation>
+    </message>
+    <message id="settings-history-cancel">
+        <source>Cancel</source>
+        <extracomment>Button label</extracomment>
+        <translation>Cancelar</translation>
+    </message>
+    <message id="settings-marker-section">
+        <source>Marker</source>
+        <extracomment>Section header</extracomment>
+        <translation>Marcador</translation>
+    </message>
+    <message id="settings-marker-slider_label">
+        <source>Mark detected code</source>
+        <extracomment>Slider label</extracomment>
+        <translation>Marcar o código detetado</translation>
+    </message>
+    <message id="settings-marker-slider_value" numerus="yes">
+        <source>%1 second(s)</source>
+        <extracomment>Marker slider value</extracomment>
+        <translation>
+            <numerusform>%1 segundo</numerusform>
+            <numerusform>%1 segundos</numerusform>
+        </translation>
+    </message>
+    <message id="history-title">
+        <source>History</source>
+        <extracomment>History page title</extracomment>
+        <translation>Historial</translation>
+    </message>
+    <message id="history-menu-clear">
+        <source>Clear</source>
+        <extracomment>Pulley menu item</extracomment>
+        <translation type="unfinished">Limpar</translation>
```

Comment:
Did you leave these "unfinished" attributes on purpose? I use it to mark translations that haven't been validated by a native speaker (e.g. those coming out of Google translate)

Reply:
Nope. Didn't pay atention to it.. Damm, I have to learn GIT...

Reply:
Well, removed pt_PT, added pt, clicked on new pullrequest, not sure this was ok...